### DISCUSSION TITLE
support for multiple auth provider

### DISCUSSION THREAD

**Discussion by shayannyc25** [OpenIDC/mod_auth_openidc#1239] [General]
hi
we have a service (to be accessed  by other services) that we want to protect with 3 auth providers . Goal is that all  and any end points in this service can be accessed with jwt tokens from any of the 3 aurh provideres. I
Iooked at the docs and wasn't sure if this is supported or how to  do it.  thanks

**Comment by zandbelt** [comment]
if you're referring to multiple OpenID Connect Providers, see https://github.com/OpenIDC/mod_auth_openidc/wiki/Multiple-Providers, if you're referring to the OAuth 2.0 Resource Server role you'll need to use https://github.com/OpenIDC/mod_oauth2 (https://github.com/OpenIDC/mod_oauth2/blob/v3.4.0/oauth2.conf#L64-L70)

**Reply by shayannyc25** [reply]
this is for providers. I looked at the doc and says that in the .client file,  we have to give client secret. but we use private key jwt file to authenticate to our auth provider, not secret.  how do we configure it for this use case?
